repository-data/application/src/main/resources/hcm-config/namespaces/hee/tabPanel.yaml

```diff
@@ -62,13 +69,25 @@ definitions:
               jcr:primaryType: frontend:pluginconfig
           /tabBody:
             jcr:primaryType: frontend:plugin
-            caption: Tab body
+            caption: Tab content
             field: tabBody
             hint: ''
             plugin.class: org.hippoecm.frontend.editor.plugins.field.NodeFieldPlugin
             wicket.id: ${cluster.id}.field
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
-              ckeditor.config.appended.json: '{ stylesSet: "styles_without_h1:./styles_without_h1.js"
+              ckeditor.config.appended.json: '{ removeButtons:"Table",stylesSet: "styles_without_h1:./styles_without_h1.js"
```

Comment:
I think `ckeditor.config.appended.json` config isn't needed here as it would be inherited from `/hippo:namespaces/hippostd/html/editor:templates/_default_/@ckeditor.config.appended.json`. But, you may need to add `ckeditor.config.overlayed.json: '{ forcePasteAsPlainText: true }'` to enforce existing `forcePasteAsPlainText` functionality.

Reply:
Fixed as part of afa76befd1f94a903705208bd6f5078c50148609 commit.